docs-new/mkdocs.yml

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+site_name: Apache Iceberg
+docs_dir: home
+
+theme:
+  name: material
+  language: en
+  logo: assets/images/iceberg-logo-icon.png
+  favicon: assets/images/favicon-96x96.png
+  features:
+    - navigation.tabs
+    - navigation.path
+    - navigation.top
+    - navigation.tracking
+    - toc.follow
+    - search.suggest
+    - search.highlight
+    - content.tabs.link
+    - content.code.copy
+    - content.code.annotate
+
+plugins:
+  - search
+  - macros:
+      include_yaml:
+        - variables.yml
+  - monorepo
+
+nav:
+  - Quickstart:
+    - Hive: hive-quickstart.md
+    - Spark: spark-quickstart.md
+  - Docs: 
+    - latest: '!include home/docs/latest/mkdocs.yml'
```

Comment:
I think that `latest` should always be the latest release, not the state of the repository. I'd recommend adding `nightly` for this content.

Reply:
Sure, but we would need to add and maintain a redirect for every single page from latest to the current version. I actually think this could be generated but this continues to add more complexity to our build. Something to consider though.

Reply:
I don't think that we need redirects. We don't have them now. We just have two copies.

Reply:
Got it, does it make sense to just grab the highest semvar number after the build and copy that directory and rename it to `latest`. Then update what I currently call `latest` to `dev`?

I like `dev` as it basically implies whenever the last build was. We could even consider adding the shortened commit hash to the label `dev (<hash>)` WDYT?

Reply:
I think `nightly` is a more obvious name that it is unreleased. I think `dev` is pretty generic and may be misinterpreted.

I'd also keep it simple with `latest`. It's better to have `latest` pointing to old docs than to have it break because of some automated change. I'd point it to the right branch specifically.

Reply:
What do you suggest when you say "point" a 307 (temp) redirect?